### PR TITLE
build(deps-dev): use new @fastify org dependencies

### PR DESCRIPTION
**Pull request by Fdawgs**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "lint": "tslint --fix -c tslint.json -t stylish -p ./tsconfig.json",
     "update": "npx npm-check -u",
     "prepublishOnly": "npm run build",
-    "pretest": "npx fastify-secure-session > secret-key",
+    "pretest": "npx @fastify/secure-session > secret-key",
```
Comment (Eomm) [inline package.json:15]: this should work in win

```suggestion
    "pretest": "npx \"@fastify/secure-session\" > secret-key",
```

**Review by mcollina — APPROVED** [review]
lgtm but CI is still failing

File: package.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
     "lint": "tslint --fix -c tslint.json -t stylish -p ./tsconfig.json",
     "update": "npx npm-check -u",
     "prepublishOnly": "npm run build",
-    "pretest": "npx fastify-secure-session > secret-key",
+    "pretest": "npx \"@fastify/secure-session\" > secret-key",
```
Comment (Eomm) [inline package.json:15]:
```suggestion
    "pretest": "npx --yes \"@fastify/secure-session\" > secret-key",
```

**Comment by mcollina** [comment]
CI is still failing

**Comment by Eomm** [comment]
I need to revive my windows pc to check it if needed 🤷🏽‍♂️

**Comment by Fdawgs** [comment]
It runs fine on my windows machine after @Eomm's change in https://github.com/fastify/fastify-flash/pull/68/commits/cea01cf420e491a2601539a054be14398c047ba1, CI was throwing an EPERM error for Windows runs in https://github.com/fastify/fastify-flash/runs/6242350895?check_suite_focus=true

**Comment by climba03003** [comment]
@Fdawgs I think the current CI Failure related to https://github.com/fastify/fastify-secure-session/pull/126
The old version of `npx` do not recognize when the command and package name mismatch.

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by climba03003** [comment]
`npx \"@fastify/secure-session\" > secret-key` run fine on `>= 12`. The problem seems to be exist on `node@10` only.
It needs explicitly specify the package and command.

**Comment by Fdawgs** [comment]
> `npx \"@fastify/secure-session\" > secret-key` run fine on `>= 12`. The problem seems to be exist on `node@10` only. It needs explicitly specify the package and command.

great thanks @climba03003!

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**